### DISCUSSION TITLE
No attribute as CTkImage

### DISCUSSION THREAD

**Discussion by Gavril2431** [TomSchimansky/CustomTkinter#2839] [Q&A]
Hi, on trying to use the customtkinter image functionality it throws an error;

**AttributeError: module 'customtkinter' has no attribute 'CTkImage'.**

I tried installing customtkinter again but the same issue pops back up
how do I get a workaround?

Thanks

**Comment by ElectricCandlelight** [comment]
Do you have a code example? Did you import PIL? 

```
import customtkinter
import os
from PIL import Image
```

**Comment by Gavril2431** [comment]
yes, i used the exact same import functions
`import customtkinter`
`import os`
`from PIL import Image`
`image_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "Some_Path")`
`logo_image = customtkinter.CTkImage(Image.open(os.path.join(image_path,'logo.png')), size=(117,20))`

**Comment by LeaDevelop** [comment]
It appears that PIL module is no longer hosted, so it can't be downloaded, experienced the following error:
`ModuleNotFoundError: No module named 'PIL'`

PyCharm returns following, using Python 3.11.2:
```
ERROR: Could not find a version that satisfies the requirement PIL (from versions: none)
ERROR: No matching distribution found for PIL
```

I used following documentation and code example from there:
https://customtkinter.tomschimansky.com/documentation/utility-classes/image

Workaround I used to display image: installed module Pillow by Jeffrey A. Clark
